src/components/VaultRow/lib/VaultDepositForm.tsx

```diff
@@ -170,7 +166,7 @@ const VaultDepositForm: FC<VaultProps> = (props) => {
       </h2>
       <FormProvider {...form}>
         <TokenForm
-          isError={prepareDeposit.isError || prepareDepositUnderlying.isError}
+          isError={prepareDeposit.isError}
```

Comment:
why prepareDepositUnderlying.isError was deleted?

Reply:
This should not have been deleted.